### ISSUE TITLE
Istio 1.5 Virtual Machines in Multi-Network Meshes example not working

### ISSUE THREAD

**Issue by FrimIdan** [istio/istio#21972]
I've followed the instructions [here](https://istio.io/docs/examples/virtual-machines/multi-network/ ) and encounter several issues:
1. The path to deploy istio control plane is missing `install/kubernetes/operator/examples/vm/values-istio-meshexpansion.yaml` -- I've used `operator/data/examples/vm/values-istio-meshexpansion-gateways.yaml`
2. After running istioctl manifest apply and setting up the VM, istio log files shows the following error:
(**34.74.31.14 is the GW IP**)
```
failed to read pod labels: open ./etc/istio/pod/labels: no such file or directory
2020-03-08T10:08:54.684334Z	info	pickfirstBalancer: HandleSubConnStateChange: 0xc000642720, {TRANSIENT_FAILURE connection error: desc = "transport: Error while dialing dial tcp 34.74.31.14:15012: i/o timeout"}
```
3. I've tried to add 15012 to the ingressgatway svc but then the following logs appear in istiod logs
`2020-03-08T11:37:43.124907Z	info	grpc: Server.Serve failed to complete security handshake from "10.48.1.26:54700": tls: first record does not look like a TLS handshake`

[ ] Configuration Infrastructure
[X ] Docs
[ ] Installation
[X ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**
No errors after applying mesh expansion multi network example

**Steps to reproduce the bug**
Follow the [instructions](https://istio.io/docs/examples/virtual-machines/multi-network/ ) 

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**
client version: c3c353285578eb68b334fc8766746b754b6b3789
control plane version: 1.5.0
data plane version: 1.5.0 (3 proxies)
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-11T18:14:22Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.12-gke.25", GitCommit:"654de8cac69f1fc5db6f2de0b88d6d027bc15828", GitTreeState:"clean", BuildDate:"2020-01-14T06:01:20Z", GoVersion:"go1.12.11b4", Compiler:"gc", Platform:"linux/amd64"}

**How was Istio installed?**
istioctl manifest apply -f operator/data/examples/vm/values-istio-meshexpansion-gateways.yaml --set values.global.tag=1.5.0 --set values.global.hub=docker.io/istio
**Environment where bug was observed (cloud vendor, OS, etc)**
GKE & EC2 VM (Canonical, Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-07-22)

**Comment by oaktowner** [comment]
Question -- does anyone know if this bug exists 1.4.6? If/when fixed, will it be backported?

**Comment by irisdingbj** [comment]
@FrimIdan Please do use ` install/kubernetes/operator/examples/vm/values-istio-meshexpansion.yaml`  as described in the doc when you install Istio.   
For the certificate issue you encountered, please make sure you have followed step about generating certs/key for VM and put them under /etc/certs in VM: https://istio.io/docs/examples/virtual-machines/single-network/

**Comment by FrimIdan** [comment]
@irisdingbj tnx for the response.
I've installed 1.5.1, follow the instructions.
I've generated certs/key for VM with the following command:
`./generate_cert -client -host spiffee://cluster.local/ns/vm/sa/vm -signer-cert certs/ca-cert.pem -signer-priv certs/ca-key.pem --out-priv key.pem --out-cert cert-chain.pem -mode signer`

The `certs/ca-cert.pem` and `certs/ca-key.pem` are from the cacerts I've created. - **this is the right way to do it?!**

All works fine, but when I've tried to encrypt connection to the VM via a destination rule the connection fails due to
`client disconnected, failure reason: TLS error: 268435581:SSL routines:OPENSSL_internal:CERTIFICATE_VERIFY_FAILED`

I've also tried to run node_agent in the VM (I've relised that it was removed from the docs)
`sudo ./node_agent --ca-address istiod.istio-system.svc:15012 --log_output_level debug`
And got the following failure:
`CSR signing failed: rpc error: code = Unauthenticated desc = request authenticate failure, no caller identity. Will retry in 5s`
